rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -333,7 +333,7 @@ public Object eth_syncing() {
     public String eth_coinbase() {
         String s = null;
         try {
-            return s = toJsonHex(minerServer.getCoinbaseAddress());
+            return s = toJsonHex(minerServer.getCoinbaseAddress().getBytes());
```

Comment:
maybe we can return `minerServer.getCoinbaseAddress().toString()` here

Reply:
These two usages we left as is to avoid changing Web3 behavior, given we don't have a comprehensive test suite.
But we could definitely encapsulate it, maybe something `RskAddress.toJsonHex()`.
Thoughts?

Reply:
I'm sorry, I misread the method call. I thought it said just `toHex`. It's ok the way it actually is.